### ISSUE TITLE
Recording randomly stops without apparent error

### ISSUE THREAD

**Issue by stonie08** [obsproject/obs-studio#3792]
Hello,

Recently my OBS 26.0.2 started having issues with recording. When using my AMD GPU as encoder, seemingly randomly the Disk "Disk full in (approx.)" field will change to a big, negative number. When I then press ""Stop Recording" the button changes to "Stopping recording..." and stays like that. Additionally the "REC:" counter keeps incrementing. If I quit OBS by pressing X, I get the dialogue asking I really want to quit because OBS is currently active. Confirming that closes the OBS window, however, it still stays in the taskbar and I have to exit it via the task-manager.
The recording is only good until the "Disk full in" stat changed it's value a negative number. This makes is especially annoying since there's no other indication that recording stopped working. Furthermore, the first time this issue appeared this also crashed a stream that was done with CPU encoding in parallel.

![2020-11-28 01_20_34-OBS 26 0 2 (64-bit, windows) - Profile_ Unbenannt - Scenes_ Unbenannt](https://user-images.githubusercontent.com/2730294/100526412-019a2980-31c9-11eb-8910-9af490f73c37.png)

**Things I tried:**
Fully Reinstalled OBS (including deleting user settings)
Downgraded AMD driver from 20.11.2 to 20.9.2 and also reinstalled 20.11.2
Used a different drive to save the recoding to

I did some tests and this might be an issue with only the GPU encoding, as I couldn't reproduce it with only the CPU encoding acitve. However, it doesn't seem to be connected to e.g.  #3767 as I updated to 20.11.2 as a result of having the issue and also rolling back to 20.9.2 didn't fix it. I also can add any bitrate number in the bitrate field.

**Comment by notr1ch** [comment]
Please use the forums or Discord for support.

**Comment by stonie08** [comment]
I did and was referred to issue #3767. However, downgrading didn't work, since this issue persists with the older 20.9.2 driver so I suspect a separate issue, which I opened.
